package/yast2-aduc.spec

```diff
@@ -17,21 +17,15 @@
 
 
 Name:           yast2-aduc
-Version:        1.2
+Version:        1.3
 Release:        0
 Summary:        Active Directory Users and Computers for YaST
 License:        GPL-3.0-only
 Group:          Productivity/Networking/Samba
-Url:            http://www.github.com/yast/yast2-aduc
+Url:            https://www.github.com/yast/yast2-aduc
```

Comment:
This is redirected to https://github.com/yast/yast2-aduc, use the shorter version to be consistent.